frontend/src/concepts/notebooks/apiHooks/useRouteForNotebook.ts

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { getRoute } from '#~/api';
+import useFetch, { FetchStateCallbackPromise, FetchStateObject } from '#~/utilities/useFetch';
+import { K8sAPIOptions } from '#~/k8sTypes';
+
+type NotebookRouteState = {
+  route: string | null;
+};
```

Comment:
We can totally remove this type. Return `<string | null>` as the result of the network call